documentation/milestones/individual_milestones/communication_platforms.md

```diff
@@ -60,31 +114,63 @@ Any code used to deploy or integrate these services should be made open source.
 
 ### Definition of done
 
-- [ ] Onboarding instructions for new users are clearly and accessibly documented in a public place.
-- [ ] Instructions for the internal team that assists with onboarding new users are clearly and accessibly documented in a public place.
-- [ ] Without logging in, members of the public can see tickets and wiki documents.
-- [ ] New users can be onboarded to chat for no cost to the user in a process that takes less than 2 days.
-- [ ] New users can be onboarded to wiki for no cost to the user in a process that takes less than 2 days.
-- [ ] Services are accessible to all people on the HHS network, White House network, public internet, and preferably most or all agency networks.
-- [ ] In chat, users can send private messages to each other as well as view private and public channels. Users can use emojis to react to messages.
-- [ ] In wiki, users can see history of changes to all documents.
-- [ ] In tickets, users can create epics (or a similar concept) and track burndown.
-- [ ] At least 5 members of HHS full-time staff are onboarded to all tools.
-- [ ] At least 5 members of contractor teams (employed by HHS) are onboarded to all tools.
-- [ ] At least 3 members of the general public are onboarded to all tools.
-- [ ] Integrate Google Analytics or similar tool into wiki-style tool.
-- [ ] Code for managing and deploying these services is deployed to `main` & PROD (if necessary)
-- [ ] Default 'templates' are created for tickets, epics, etc, that have the fields we've decided are necessary for the project.
-- [ ] [to be added - definition of done for calendar, email listserves, email inboxes]
+- [ ] The following conditions have been satisfied for all tools:
+  - [ ] At least 5 members of HHS full-time staff have been onboarded.
+  - [ ] At least 5 members of contractor teams (employed by HHS) have been onboarded.
+  - [ ] At least 3 members of the general public have been onboarded.
+  - [ ] New users can be onboarded to the tool for no cost to the user in a process that takes less than 2 days.
+  - [ ] Services are accessible to all people on the HHS network, White House network, public internet, and preferably most or all agency networks.
```

Comment:
let's switch white house to a nice-to-have? It can be really hard to get WH access.

Reply:
@lucasmbrown-usds Sure thing! I rephrased this to:

> Services are accessible to all people on the HHS network, public internet, and *preferably* the White House network and most or all agency networks

Let me know if that updated acceptance criteria works or if I should strike White House network access altogether from this bullet point